reporter/bin/check.py

```diff
@@ -55,6 +56,9 @@
 # @see https://kibana.wikia-inc.com/#/dashboard/elasticsearch/Phalanx%20service%20logs
 reports += PhalanxSource().query(threshold=5)
 
+# @see https://kibana.wikia-inc.com/#dashboard/temp/AVOjc8W-tlVOnUfDvyCL
```

Comment:
I thought that Kibana temporary dashboards are pruned regularly. Does it make sense to put it here then?